### PR TITLE
Add log support

### PR DESCRIPTION
**Pull request by genestack-solomatin**
Add `--show-logs` option to print log for each request

### CODE REVIEW

File: genestack_client/genestack_connection.py [hunk 1]
```diff
@@ -208,6 +213,17 @@ def __invoke(self, path, to_post):
                 debug=self.connection.debug,
                 stack_trace=response.get('errorStackTrace')
             )
+
+        logs = response['log']
+        if logs and (self.connection.show_logs or self.connection.debug):
+            message = '\nLogs:\n'
```
Comment (klas-genestack) [inline genestack_client/genestack_connection.py]:
```python
message = '\nLogs:\n' + '\n'.join(item['message'] + item.get('stackTrace', '') for item in logs)
```

File: genestack_client/settings/genestack_user.py [hunk 2]
```diff
@@ -59,10 +59,14 @@ def get_connection(self, interactive=True, debug=False):
 
         :param interactive: ask email and/or password interactively.
         :type interactive: bool
+        :param debug: print stack-trace in case of exception
```
Comment (klas-genestack) [inline genestack_client/settings/genestack_user.py]: "stack trace"

File: genestack_client/settings/genestack_user.py [hunk 3]
```diff
@@ -59,10 +59,14 @@ def get_connection(self, interactive=True, debug=False):
...
+        :type debug: bool
+        :param show_logs: print logs received from server
```
Comment (klas-genestack) [inline genestack_client/settings/genestack_user.py]: "print application logs (received from server)"

File: genestack_client/genestack_connection.py [hunk 4]
```diff
@@ -44,9 +44,13 @@ class Connection:
     A class to handle a connection to a specified Genestack server.
     Instantiating the class does mean you are logged in to the server.
     To do so, you need to call the :py:meth:`~genestack_client.Connection.login` method.
+
+    Connection support retrieving debug information.
+      - ``debug`` will print additional traceback from application
+      - ``show_logs``  allow
```
Comment (klas-genestack) [inline genestack_client/genestack_connection.py]: Hm, what?

File: genestack_client/utils.py [hunk 5]
```diff
@@ -48,6 +48,7 @@ def make_connection_parser(user=None, password=None, host=None):
     group.add_argument('-u', dest='user', metavar='<user>', default=user, help='user alias from settings or email')
     group.add_argument('-p', dest='pwd', default=password, metavar='<password>', help='user password')
     group.add_argument('--debug', dest='debug', help='print additional stacktrace on error', action='store_true')
+    group.add_argument('--show-logs', dest='show_logs', help='print logs created with ApplicationLogger', action='store_true')
```
Comment (klas-genestack) [inline genestack_client/utils.py]: Let's be consistent: `help="print application logs (received from server)"`

**Comment by genestack-solomatin** [comment]
udated

**Pull request merged**